source/documentation/how-to-release.html.md

```diff
@@ -68,7 +69,7 @@ Closing a staging repository makes the binaries available in [staging][staging],
 #### 4. Run the source tarball script
 
 ```
-sh dev/source-release.sh <version> <rc-number>
+dev/source-release.sh <version> <rc-number>
```

Comment:
dev/source-release.sh is not executable in my repo, it doesn't work without `sh` or `.`

Reply:
My bad, didn't notice the corresponding change in the other PR.